src/hera/workflows/cron_workflow.py

```diff
@@ -90,40 +137,44 @@ def build(self) -> TWorkflow:
         """Builds the CronWorkflow and its components into an Argo schema CronWorkflow object."""
         self = self._dispatch_hooks()
 
-        return _ModelCronWorkflow(
-            api_version=self.api_version,
-            kind=self.kind,
-            metadata=ObjectMeta(
-                annotations=self.annotations,
-                cluster_name=self.cluster_name,
-                creation_timestamp=self.creation_timestamp,
-                deletion_grace_period_seconds=self.deletion_grace_period_seconds,
-                deletion_timestamp=self.deletion_timestamp,
-                finalizers=self.finalizers,
-                generate_name=self.generate_name,
-                generation=self.generation,
-                labels=self.labels,
-                managed_fields=self.managed_fields,
-                name=self.name,
-                namespace=self.namespace,
-                owner_references=self.owner_references,
-                resource_version=self.resource_version,
-                self_link=self.self_link,
-                uid=self.uid,
-            ),
+        model_workflow = _ModelCronWorkflow(
+            metadata=ObjectMeta(),
             spec=CronWorkflowSpec(
-                concurrency_policy=self.concurrency_policy,
-                failed_jobs_history_limit=self.failed_jobs_history_limit,
-                schedule=self.schedule,
-                starting_deadline_seconds=self.starting_deadline_seconds,
-                successful_jobs_history_limit=self.successful_jobs_history_limit,
-                suspend=self.cron_suspend,
-                timezone=self.timezone,
-                workflow_metadata=None,
+                schedule="",
```

Comment:
```suggestion
                schedule=self.schedule,
```

Reply:
Haha yes that makes more sense as a default value! Then the annotation for `schedule` isn't strictly needed 🤔

Reply:
Turns out this helped me realise there's a bug with this - `CronWorkflow._from_model(w.build())` - this will fail because the default init for a CronWorkflow needs a value for the schedule, so the line `hera_obj = cls()` fails. Making a change for this